### ISSUE TITLE
crash on user deletion from admin page

### ISSUE THREAD

**Issue by wceichelb** [OHSU-FM/reindeer#24]
to reproduce:

delete any `user` with a `dashboard` that has at least one associate `dashboard_widget`

the `user` deletion goes through, but orphan `dashboard_widget`s are left in the db, which also then causes a crash on visiting the `/admin/dashboard_widget` route.

**Comment by hattwj** [comment]
Sounds good! A little hard to follow with the white space noise. If we want to strip trailing characters should we open a seperate ticket?

**Comment by wceichelb** [comment]
yeah i have it set someplace in my dotfiles to automatically strip whitespace on write. in this case, though, there're only two changed lines. i'll open a bug/look into stripping whitespace otherwise.
